### DISCUSSION TITLE
filter windows in table is not resizable

### DISCUSSION THREAD

**Discussion by niepi** [grafana/grafana#53828] [Dashboards, Panels & Visualizations]
we using the option to "Filter by Values" in tables and the values are very long, it makes hard to navigate this view.

Suggestion:

can you make this window resizable, for better view of the data

Grafana version: 8.5.5

![thumbnail_image001](https://user-images.githubusercontent.com/4350/185124768-3783fefd-132d-4aca-92e3-932082fec02f.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
